### ISSUE TITLE
Lexicon - Aim for enhanced AAA target sizes

### ISSUE THREAD

**Issue by catbusofthecanals** [jadu/pulsar#1589]
**Describe the bug**
The target size of many interactive elements on the site falls below the enhanced AAA requirements of 2.5.5 Target Size (Enhanced). While not a requirement at a AA level of WCAG conformance, where possible, AAA target sizes should be strived for to make content as operable as possible for all.

The areas where target sizes are below the AAA requirement include:

Buttons with class=”btn” (Button examples under Form Helpers/Menu/Save) - 38 px in height
Toolbar dropdown links (Pulsar City Council) - 36 px in height
Radio buttons and Checkbox inputs - 22 x 22px in height 

**To Reproduce**
1. Go to Pulsar Lexicon
2. Click on Form Helpers > Normal Buttons
3. Inspect button elements to see size

**Expected behavior**
Where possible, as many interactive elements should meet 2.5.5 Target Size (Enhanced) requirements of 44 x 44 px.

**Screenshots**
![Image](https://github.com/user-attachments/assets/3d4cc816-688b-4acb-ae90-14b4cbd60b4d)

**Additional context**
Related success criteria:
[2.5.5 Target Size (Enhanced) (AAA)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced)

**Suggested fixes**
Slightly increase the target areas in as many areas listed as possible until the height and width of the target itself is at least 44 x 44 px.
